recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+            "msvc": "191",
+            "Visual Studio": "15",
+        }
+
+    @property
+    def _has_sse_avx(self):
+        return self.settings.arch in ["x86", "x86_64"]
+
+    @property
+    def _has_neon(self):
+        return "arm" in self.settings.arch
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.settings.os != "Emscripten":
+            self.requires("onetbb/2021.12.0")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+        compiler_version = Version(self.settings.compiler.version)
+
+        # See https://github.com/RenderKit/embree/blob/master/CMakeLists.txt#L538
+        if self.settings.compiler == "apple-clang" and not self.options.shared and compiler_version >= "9.0":
+            raise ConanInvalidConfiguration(f"{self.ref} static with apple-clang >=9 and multiple ISA (simd) is not supported")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["EMBREE_STATIC_LIB"] = not self.options.shared
+        tc.variables["BUILD_TESTING"] = False
+        tc.variables["EMBREE_INSTALL_DEPENDENCIES"] = False
+        tc.variables["EMBREE_TUTORIALS"] = False
+        tc.variables["EMBREE_BACKFACE_CULLING"] = False 
+        tc.variables["EMBREE_IGNORE_INVALID_RAYS"] = False
+        tc.variables["EMBREE_ISPC_SUPPORT"] = False
+        tc.variables["EMBREE_TASKING_SYSTEM"] = "INTERNAL" if is_apple_os(self) or self.settings.os == "Emscripten" else "TBB"
+        tc.variables["EMBREE_MAX_ISA"] = "NONE"
+        tc.variables["EMBREE_ISA_NEON"] = self._has_neon
+        tc.variables["EMBREE_ISA_NEON2X"] = self._has_neon
+        tc.variables["EMBREE_ISA_SSE2"] = self._has_sse_avx
+        tc.variables["EMBREE_ISA_SSE42"] = self._has_sse_avx
+        # For Emscripten disable TBB and all ISAs. It will compile only for SSE
+        if self.settings.os == "Emscripten":
+            tc.variables["EMBREE_ISA_AVX"] = self._has_sse_avx
+            tc.variables["EMBREE_ISA_AVX2"] = self._has_sse_avx
+            tc.variables["EMBREE_ISA_AVX512"] = self._has_sse_avx and not is_msvc(self)
+            if is_msvc(self):
+                tc.variables["USE_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
```

Comment:
```suggestion
        if is_msvc(self):
            tc.variables["USE_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
```
Reduce indentation to cover method scope.